tests/samples/ns1-pkg2/ns1-pkg2.toml

```diff
@@ -0,0 +1,9 @@
+[build-system]
+requires = ["flit"]
+
+[tool.flit.metadata]
+module = "ns1.pkg2"
+author = "Sir Robin"
+author-email = "robin@camelot.uk"
+home-page = "http://github.com/sirrobin/module1"
```

Comment:
sirrobin is a real github account!

Probably better to use canonical `example.com` domain for these

Reply:
Yes, that's a good idea. I think the 'Sir Robin' example is copied and pasted around the tests.